mne_connectivity/decoding/decomposition.py

```diff
@@ -289,11 +292,13 @@ def fit(self, X, y=None):
         csd = self._compute_csd(X)
 
         # instantiate connectivity estimator and add CSD information
-        self._conn_estimator = self._conn_estimator(
+        self._conn_estimator = self._conn_estimator_class(
             n_signals=X.shape[1],
             n_cons=1,
             n_freqs=1,
             n_times=0,
+            n_components=self.n_components,
+            store_con=False,
             store_filters=True,
             n_jobs=self.n_jobs,
         )
```

Comment:
Fixes a bug where `.fit()` could not be called multiple times on the same object